### PR TITLE
Update builds.md

### PR DESCRIPTION
**Pull request by paulvi**
please review after PR #78 (renaming)

BTW, the title may be better to rename to just "Builds". (And change in main menu as well)

### CODE REVIEW

File: source/builds.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Accordingly, the source may not always compile or be in a release-ready state.
 
 *Access at your own risk!*
 
-NOTE: The full source code is provided with each [release.](downloads.html) If you simply want to build
+NOTE: The full source code is provided with each [release](/download). If you simply want to build
```
Comment (lukaszlenart) [inline source/builds.md]: `.cgi`?

File: source/builds.md [hunk 2]
```diff
@@ -112,4 +112,4 @@ Unless required by applicable law or agreed to in writing, software distributed
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations under the License.
 
-Next: [Release Guidelines](releases.html)
+Next: [Release Guidelines](release-guidelines)
```
Comment (lukaszlenart) [inline source/builds.md]: `.html`?

File: source/builds.md (hunk 1 above)
Comment (paulvi) [reply source/builds.md]: done

File: source/builds.md (hunk 2 above)
Comment (paulvi) [reply source/builds.md]: intentionally not adding extension.

Maybe one day /release-guidelines will be a folder, or `.jsp`.  
Would you accept such approach?

Comment (lukaszlenart) [reply source/builds.md]: Why folder? And not `.jsp` for sure :)

Comment (paulvi) [reply source/builds.md]: ... the point was missed. Just adding `.html`

**Comment by lukaszlenart** [comment]
👍

**Pull request merged**